packages/dropdownList/src/Default/Default.tsx

```diff
@@ -0,0 +1,210 @@
+
+    const downHandler = ({ key }: { key: string }) => {
+      if (key === targetKey) {
+        setKeyPressed(true);
+      }
+    };
+
+    const upHandler = ({ key }: { key: string }) => {
+      if (key === targetKey) {
+        setKeyPressed(false);
+      }
+    };
+
+    React.useEffect(() => {
+      window.addEventListener('keydown', downHandler);
+      window.addEventListener('keyup', upHandler);
+
+      return () => {
+        window.removeEventListener('keydown', downHandler);
+        window.removeEventListener('keyup', upHandler);
+      };
+    });
+    return keyPressed;
+  };
+
+  const downPress = useKeyPress('ArrowDown');
+  const upPress = useKeyPress('ArrowUp');
+  const enterPress = useKeyPress('Enter');
+  const tabPress = useKeyPress('Tab');
+
+  React.useEffect(() => {
+    if (dropdown?.items.length && downPress) {
+      if (!dropdownOpen) {
+        setDropdownOpen(true);
+      }
+      if (dropdown.items.length) {
+        setCursor(prevState =>
+          typeof dropdown?.items?.length === 'number' &&
+          prevState < dropdown?.items?.length - 1
+            ? prevState + 1
+            : prevState
+        );
+      }
+    }
+    setSelected(
+      dropdown?.items[cursor as unknown as keyof DropDownContentItemProps]
+    );
+    const selectedItem = document.querySelector(
+      '.owlui-dropdown-menu-selectedMenuItem'
+    );
+    selectedItem?.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
+  }, [downPress]);
+  React.useEffect(() => {
+    if (dropdown?.items.length && upPress) {
+      if (!dropdownOpen) {
+        setDropdownOpen(true);
+      }
+      setCursor(prevState => (prevState > 0 ? prevState - 1 : prevState));
+    }
+    setSelected(
+      dropdown?.items[cursor as unknown as keyof DropDownContentItemProps]
+    );
+    const selectedItem = document.querySelector(
+      '.owlui-dropdown-menu-selectedMenuItem'
+    );
+    selectedItem?.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
+  }, [upPress]);
+  React.useEffect(() => {
+    if (dropdown?.items.length && enterPress) {
+      setSelected(
+        dropdown?.items[cursor as unknown as keyof DropDownContentItemProps]
+      );
+    }
+  }, [cursor, enterPress]);
+  React.useEffect(() => {
+    if (dropdown?.items.length && tabPress) {
+      if (dropdownOpen) {
+        setDropdownOpen(false);
+      }
+    }
+  }, [tabPress]);
+  const handleClickOutside = (ref: React.MutableRefObject<HTMLDivElement>) => {
+    React.useEffect(() => {
+      const closeList = (event: MouseEvent) => {
+        if (ref.current && !ref.current.contains(event.target as Node)) {
+          setDropdownOpen(false);
+        }
+      };
+      document.addEventListener('mousedown', closeList);
+      return () => {
+        document.removeEventListener('mousedown', closeList);
+      };
+    }, [ref]);
+  };
+
+  // type itemType = { id: number; name: string };
+
+  const ListItem = ({
```

Comment:
Ideally, we would move this to its own component, kinda like what the `card` component looks like.

To create the new sub-component you can use the create option:
```
yarn run create:component -o dropdownList@listItem
```